### DISCUSSION TITLE
NeoTree not refreshing on project switch

### DISCUSSION THREAD

**Discussion by niklaswallerstedt** [LazyVim/LazyVim#2064] [Q&A]
Just started with LazyVim and very impressed so far 🥳 One issue that I have, that I have not been able to find a solution for is when using `extras.util.project` from the LazyExtras.

Whenever I switch to a file from a different project, NeoTree is not updating until after a toggle of the explorer. Is there some setting to fix this or can I hook into an event on "project switch" to perform the refresh of Neotree myself?

**Comment by OrestesK** [comment]
> Just started with LazyVim and very impressed so far 🥳 One issue that I have, that I have not been able to find a solution for is when using `extras.util.project` from the LazyExtras.
> 
> Whenever I switch to a file from a different project, NeoTree is not updating until after a toggle of the explorer. Is there some setting to fix this or can I hook into an event on "project switch" to perform the refresh of Neotree myself?

Ever end up finding a solution?

**Reply by niklaswallerstedt** [reply]
No unfortunately not - forgot about this one, haven't been using the projects feature for some time. Using tmux sessions for each project now instead.
